### PR TITLE
add meetup page

### PR DESCRIPTION
**Pull request by cstnaya**
create a meetup page

### CODE REVIEW

File: content/pages/meetup/insex.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+======
+meetup
+======
+
+:date: 2022-09-08
+:url:
+:save_as: meetup/index.html
+
```
Comment (yungyuc) [inline content/pages/meetup/insex.rst:8]: Could you please add a simple sentence here, like "Meetups are to be added."?  A totally empty page would look a little bit strange for a public site.

**Review by yungyuc — CHANGES_REQUESTED** [review]
LGTM.  Could you help make a minor change?

File: content/pages/meetup/insex.rst (hunk 1 above)
Comment (cstnaya) [reply content/pages/meetup/insex.rst:8]: OK!! ('ω')ノ
I also added some new lines, so that the page wouldn't look weird.

Comment (yungyuc) [reply content/pages/meetup/insex.rst:8]: It looks nice!  Thanks a lot.  You really know how to make web pages beautiful.

**Comment by yungyuc** [comment]
This solves #129

**Pull request merged**